kDriveShareExtension/ShareNavigationViewController.swift

```diff
@@ -17,21 +17,20 @@
  */
 
 import UIKit
-import InfomaniakCore
+import InfomaniakCoreUI
 import InfomaniakLogin
 import kDriveCore
+import InfomaniakDI
 
 class ShareNavigationViewController: TitleSizeAdjustingNavigationController {
 
-    private var accountManager: AccountManager!
+    @InjectService var accountManager: AccountManageable
```

Comment:
We are in an extension so we need to setup DI from scratch (Same for FileProviderExtension)

Reply:
I moved the `FactoryService` code to kDriveCore, so I could use it from all the extensions.

We will probably need a mechanism that allows for setting up Factories in kDrive _and_ kDriveCore, but for now this is enough.